SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/EntityComponentRecordTest.cpp

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+
+#include "SpatialView/EntityComponentRecord.h"
+
+#include "EntityComponentTestUtils.h"
+
+#define ENTITYCOMPONENTRECORD_TEST(TestName) \
+	GDK_TEST(Core, EntityComponentRecord, TestName)
+
+using namespace SpatialGDK;
```

Comment:
I believe it's acceptable to use `using namespace` in .cpp files. We have multiple .cpp files do it (mostly Test files though).

Reply:
Oh that is true, cpp should be fine

Reply:
Commented on the other comment about this. Their use is contraversial although it does exist in the rest of the gdk. I'd be much happier with specifically using types rather than the whole namespace.
However can we just put test in the namespace. Unless there is some reason that doesn't work there isn't really any reason to have the tests in global space.

Reply:
Removed `using`, moved test code to SpatialGDK namespace instead.